site/docs/relations/logical_relations.md

```diff
@@ -338,11 +338,11 @@ The fetch operation eliminates records outside a desired window. Typically corre
 
 ### Fetch Properties
 
-| Property | Description                                                           | Required                 |
-| -------- | --------------------------------------------------------------------- | ------------------------ |
-| Input    | A relational input, typically with a desired orderedness property.    | Required                 |
-| Offset   | A non-negative integer. Declares the offset for retrieval of records. | Optional, defaults to 0. |
-| Count    | A non-negative integer or -1. Declares the number of records that should be returned. -1 signals that ALL records should be returned. | Required |
+| Property          | Description                                                                                                                                                            | Required                 |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------ |
+| Input             | A relational input, typically with a desired orderedness property.                                                                                                     | Required                 |
+| Offset Expression | An expression which evaluates to a non-negative integer. Declares the offset for retrieval of records.                                                                 | Optional, defaults to 0. |
+| Count Expression  | An expression which evaluates to a non-negative integer or -1. Declares the number of records that should be returned. -1 signals that ALL records should be returned. | Required |
```

Comment:
Nit: pad last column with spaces?

Reply:
Fixed (though it appears padding is not applied consistently throughout the documentation).